docs/content/concepts/resources.mdx

```diff
@@ -125,28 +142,21 @@ defs = Definitions(
 
-</TabItem>
-</TabGroup>
-
-There are many supported config types that can be used when defining resources. See the [advanced config types documentation](/concepts/configuration/advanced-config-types) for a more comprehensive overview on the available config types.
+Ops specify resource dependencies by annotating the resource as a parameter to the op function.
 
-### Using resources with sensors
+To provide resource values to ops, attach them to the <PyObject object="Definitions" /> objects. These resources are automatically passed to the function at runtime.
 
-[Sensors](/concepts/partitions-schedules-sensors/sensors) can use resources in the same way as ops and assets, which can be useful for querying external services for data.
-
-To specify resource dependencies on a sensor, annotate the resource type as a parameter to the sensor's function. For more information and examples, refer to the [Sensors documentation](/concepts/partitions-schedules-sensors/sensors#using-resources-in-sensors).
+There are many supported config types that can be used when defining resources. Refer to the [advanced config types documentation](/concepts/configuration/advanced-config-types) for a more comprehensive overview of the available config types.
 
-### Using resources with schedules
-
-[Schedules](/concepts/partitions-schedules-sensors/schedules) can also use resources in case your schedule logic needs to interface with an external tool, or to make your schedule logic more testable.
+---
 
-To specify resource dependencies on a schedule, annotate the resource type as a parameter to the schedule's function. For more information and examples, refer to the [Schedules documentation](/concepts/partitions-schedules-sensors/schedules#using-resources-in-schedules).
+## Configuring resources
 
 ### Using environment variables with resources
 
-Resources can be configured using environment variables, which is useful for secrets or other environment-specific configuration. If you're using Dagster Cloud, environment variables can be [configured directly in the UI](/dagster-cloud/managing-deployments/environment-variables-and-secrets).
+Resources can be configured using environment variables, which is useful for secrets or other environment-specific configuration. If you're using [Dagster Cloud](/dagster-cloud), environment variables can be [configured directly in the UI](/dagster-cloud/managing-deployments/environment-variables-and-secrets).
 
-To use environment variables, pass an <PyObject object="EnvVar" /> when constructing your resource. `EnvVar` inherits from `str` and can be used to populate any string config field on a resource. The value of the environment variable will be evaluated at launch time.
+To use environment variables, pass an <PyObject object="EnvVar" /> when constructing the resource. `EnvVar` inherits from `str` and can be used to populate any string config field on a resource. The value of the environment variable will be evaluated at launch time.
 
```

Comment:
A really common support question we get is why to use `EnvVar` instead of `os.getenv()` - maybe we can add a blurb here to explain the difference? I'm pretty sure it has to do with when the value is pulled from the environment - `os.getenv()` would get read when the code location is loaded, whereas `EnvVar` would get read when the run is actually launched. There might also be some privacy implications in the UI (env vars read with `os.getenv()` might get displayed). 

@benpankow  can you confirm if the above is correct for `EnvVar` vs `os.getenv()`?

Reply:
i can confirm this is correct. let's add that explainer in.